.github/workflows/deploy-changed-cf.yaml

```diff
@@ -1,5 +1,15 @@
 name: Deploy changed CloudFormation templates
 on:
+  workflow_dispatch:
+    inputs:
+      cluster:
+        type: choice
+        description: Select environment to deploy to
+        options:
+        - staging
+        - production
+        - staging and production
+        default: staging
   push:
```

Comment:
didn't you do these changes and merge them as part of a different PR?

Reply:
I added them here so that I can trigger this branch manually